apps/app/locals.tf

```diff
@@ -13,6 +13,31 @@
 #  limitations under the License.
 
 locals {
+  repository_name = var.app_id
+
   feature_store_odbc = "Driver={ODBC Driver 18 for SQL Server};Server=tcp:${data.azurerm_mssql_server.feature_store.fully_qualified_domain_name},1433;Database=${var.feature_store_db_name};Authentication=ActiveDirectoryMsi;Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;"
-  is_prod            = length(regexall("prod", lower(var.environment))) > 0
+  acr_repository     = var.app_id
+  create_repo        = var.app_config.managed_repo != null
+
+  staging_slot_name   = "staging"
+  core_gh_env         = var.environment
+  core_branch_name    = local.core_gh_env
+  staging_gh_env      = var.app_config.add_staging_slot ? "${var.environment}-testing_slot" : null
+  staging_branch_name = local.staging_gh_env
+  branches_and_envs = var.app_config.add_staging_slot ? {
+    "${local.core_branch_name}"    = local.core_gh_env
+    "${local.staging_branch_name}" = local.staging_gh_env
+  } : { "${local.core_branch_name}" = local.core_gh_env }
+
+  acr_deploy_reusable_workflow_filename = "acr_deploy_reusable.yml"
+  slot_swap_reusable_workflow_filename  = "slot_swap_reusable.yml"
+
+  envs_and_workflow_templates = var.app_config.add_staging_slot ? {
+    "${local.core_gh_env}"    = data.template_file.core_github_workflow
+    "${local.staging_gh_env}" = data.template_file.staging_github_workflow[0]
+  } : { "${local.core_gh_env}" = data.template_file.core_github_workflow }
+
+  webapp_name_in_webhook   = var.app_config.add_staging_slot ? "${lower(azurerm_linux_web_app.app.name)}-staging" : lower(azurerm_linux_web_app.app.name)
```

Comment:
What's the reason for the `-staging` suffix here?

Reply:
nice spot - thanks. left over from when there was webhooks on both